### PR TITLE
Update guides of kpt fn export

### PR DESCRIPTION
**Pull request by ZhuGongpu**

### CODE REVIEW

File: site/content/en/guides/consumer/function/export/cloud-build/_index.md [hunk 1]
```diff
@@ -19,18 +19,16 @@ mkdir function-export-example
...
+**All commands must be run at the root of this directory**.
 
 Use `kpt pkg get` to fetch source files of this tutorial:
 
 ```shell script
-kpt pkg get https://github.com/GoogleContainerTools/kpt/package-examples/function-export-blueprint exmaple-package
-# Init git
-git init
-git remote add origin https://github.com/<USER>/<REPO>.git
+# Fetch source files
```
Comment (Liujingfang1) [inline site/content/en/guides/consumer/function/export/cloud-build/_index.md:27]: The steps `git init` and `git remote add` are all kept in other workflow example. Why they are removed for cloudbuild?

File: site/content/en/guides/consumer/function/export/github-actions/_index.md [hunk 2]
```diff
@@ -82,11 +92,22 @@ jobs:
           - name: Run all kpt functions
             uses: docker://gcr.io/kpt-dev/kpt:latest
             with:
-                args: fn run exmaple-package
+                args: fn run example-package
+          
+          - name: Show diff
```
Comment (Liujingfang1) [inline site/content/en/guides/consumer/function/export/github-actions/_index.md:97]: Is the diff step optional? In other words, does `export` command generate it?

File: site/content/en/guides/consumer/function/export/tekton/_index.md [hunk 3]
```diff
@@ -19,18 +21,19 @@ mkdir function-export-example
...
-All commands must be run at the root of this directory.
+**All commands must be run at the root of this directory**.
 
 Use `kpt pkg get` to fetch source files of this tutorial:
 
 ```shell script
-kpt pkg get https://github.com/GoogleContainerTools/kpt/package-examples/function-export-blueprint exmaple-package
 # Init git
 git init
 git remote add origin https://github.com/<USER>/<REPO>.git
```
Comment (prachirp) [inline site/content/en/guides/consumer/function/export/tekton/_index.md:37]: One idea I've seen used in other docs is to define env variables for USER and REPO at the beginning. Later commands that use $USER and $REPO can be directly copy/pasted and run correctly.

File: site/content/en/guides/consumer/function/export/tekton/_index.md [hunk 4]
```diff
@@ -10,7 +10,9 @@ In this tutorial, you will pull an example blueprint that declares Kubernetes re
 
 ## Before you begin
 
-Before diving into the following tutorial, you may need to create a public repo on GitHub if you don't have one yet, e.g. `function-export-example`.
+*New to Tekton? Here is a [quickstart](https://tekton.dev/docs/getting-started/)*.
```
Comment (prachirp) [inline site/content/en/guides/consumer/function/export/tekton/_index.md:17]: Rest of the kpt docs keep all links at the bottom of the doc which makes it easy to change many links at once. Let's follow this convention.

This applies wherever you've added links.

File: site/content/en/guides/consumer/function/export/jenkins/_index.md [hunk 5]
```diff
@@ -54,13 +63,13 @@ curl -fsSL https://get.docker.com -o get-docker.sh
 sudo sh get-docker.sh
 ```
 
-1.  Add the `jenkins` user to the `docker` group so that docker commands can be run in Jenkins pipelines.
+2.  Add the `jenkins` user to the `docker` group so that docker commands can be run in Jenkins pipelines.
```
Comment (prachirp) [inline site/content/en/guides/consumer/function/export/jenkins/_index.md:66]: Undo these changes, it's best practice to keep "1." for all steps.

File: site/content/en/guides/consumer/function/export/tekton/_index.md [hunk 6]
```diff
@@ -8,9 +8,15 @@ description: >
 
 In this tutorial, you will pull an example blueprint that declares Kubernetes resources and two kpt functions. Then you will export a pipeline that runs the functions against the resources on [Tekton](https://tekton.dev/) and modify it to make it fully functional. How to setting up Tekton is also included if you don't have one running yet. This tutorial takes about 20 minutes.
 
+{{% pageinfo color="info" %}}
+A kpt version `v0.32.0` or higher is required.
+{{% /pageinfo %}}
+
 ## Before you begin
```
Comment (prachirp) [inline site/content/en/guides/consumer/function/export/tekton/_index.md:16]: Explicitly call out that you need `tkn` installed before you begin and link to the release page.

File: site/content/en/guides/consumer/function/export/jenkins/_index.md (hunk 5 above)
Comment (ZhuGongpu) [reply site/content/en/guides/consumer/function/export/jenkins/_index.md:66]: Here is the only case that they are not rendered correctly.

File: site/content/en/guides/consumer/function/export/cloud-build/_index.md (hunk 1 above)
Comment (ZhuGongpu) [reply site/content/en/guides/consumer/function/export/cloud-build/_index.md:33]: `gcloud builds submit .` is used to submit a build. So a git repo is not required for Cloud Build.

File: site/content/en/guides/consumer/function/export/github-actions/_index.md (hunk 2 above)
Comment (ZhuGongpu) [reply site/content/en/guides/consumer/function/export/github-actions/_index.md:103]: Technically speaking, a `diff` step is not the duty of `kpt fn export`. It is the user's call whether to add the step **manually** or not.

File: site/content/en/guides/consumer/function/export/jenkins/_index.md (hunk 5 above)
Comment (ZhuGongpu) [reply site/content/en/guides/consumer/function/export/jenkins/_index.md:66]: It is because the code blocks. I will fix them.

File: site/content/en/guides/consumer/function/export/tekton/_index.md (hunk 3 above)
Comment (ZhuGongpu) [reply site/content/en/guides/consumer/function/export/tekton/_index.md:38]: The idea is really great. But in my case, the two env variables are only used once per tutorial.
One exception is that the generated Tekton pipeline needs the link to the repo. But it is in the file content, and not easy to read from env variables.
So I prefer to keep it simple.
